fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/synchronizedsession/SynchronizedSessionRunner.java

```diff
@@ -163,6 +167,28 @@ public void endSession() {
         underlying.asyncToSync(FDBStoreTimer.Waits.WAIT_END_SYNC_SESSION, endSessionAsync());
     }
 
+    /**
+     * Releases the lock to end any synchronized session on it, no matter the current session holds the lock or not.
+     * <p>
+     * A {@link SynchronizedSessionRunner} or {@link SynchronizedSession} is not necessary for this purpose, as one can
+     * simply use the static version {@link SynchronizedSession#forceReleaseLock(Transaction, Subspace)}.
```

Comment:
I'm not sure I totally understand this comment. Is it just trying to say that one could (instead of this method)? I think it's confusing, as it almost reads as a description of how the method is implemented, which is not quite what I think the intention was.

Reply:
My intent was: A lock subspace is sufficient. Users don't have to create a sync session or a sync session runner to stop sessions.

Simplified this note to a `@see`.